### PR TITLE
Better exception handling.

### PR DESCRIPTION
**Pull request by Matt3o12**
A exception will now throw when a incorrect email or password is given

### CODE REVIEW

File: tests.py [hunk 1]
```diff
@@ -2,16 +2,17 @@
...
 
-
-class TestMega(unittest.TestCase):
+class TestMega(unittest.TestCase):    
     def setUp(self):
         unittest.TestCase.setUp(self)
-        email = os.environ.get('MEGAEMAIL') or sys.argv[0]
-        password = os.environ.get('MEGAPASSWORD') or sys.argv[1]
-        self.api_logged = Mega.from_credentials(email, password)
+        self._email = "matteo.kloiber@me.com";
```
Comment (juanriaza) [inline tests.py]: Please keep the enviroment vars. Travis-ci populate email&password from the values provided at `.travis.yml`.

File: mega/mega.py [hunk 2]
```diff
@@ -70,6 +70,9 @@ def login_ephemeral(self):
         self._login_common(res, random_password_key)
 
     def _login_common(self, res, password):
+        if (res == -2) or (res == -9):
```
Comment (juanriaza) [inline mega/mega.py]: `if res in (-2, -9)`

**Comment by Matt3o12** [comment]
Tranks for your improvements (I forgot to change it). 
I'm going to change those things and create a pull request again, ok?

**Comment by juanriaza** [comment]
It's not necessary to create a new PR. Just commit those bits and they will
appear here.

**Comment by Matt3o12** [comment]
2 days ago, I fixed those issues and you should have become a notification.

**Pull request merged**